api/src/pdc/services/policy/engine/policies/20240901_SiouleLimagne_2024_09.ts

```
@@ -0,0 +1,125 @@
+    ctx: StatelessContextInterface,
+  ) {
+    try {
+      isOperatorOrThrow(
+        ctx,
+        getOperatorsAt(this.operators, ctx.carpool.datetime),
+      );
+      onDistanceRangeOrThrow(ctx, { min: 2_000, max: 80_000 });
+      isOperatorClassOrThrow(ctx, ["B", "C"]);
+      isAdultOrThrow(ctx);
+    } catch (e) {
+      throw e;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)